core-ui/src/components/Extensibility/components/EventList.js

```diff
@@ -0,0 +1,167 @@
+import { useTranslation } from 'react-i18next';
+import { Icon } from 'fundamental-react';
+import pluralize from 'pluralize';
+import { resources } from 'resources';
+
+import { ReadableCreationTimestamp } from 'shared/components/ReadableCreationTimestamp/ReadableCreationTimestamp';
+import { Tooltip } from 'shared/components/Tooltip/Tooltip';
+import { useMessageList, EVENT_MESSAGE_TYPE } from 'hooks/useMessageList';
+
+import { EMPTY_TEXT_PLACEHOLDER } from 'shared/constants';
+
+import { ResourcesList } from 'shared/components/ResourcesList/ResourcesList';
+import { prettifyKind } from 'shared/utils/helpers';
+import { useGetTranslation } from '../helpers';
+import { jsonataWrapper } from '../helpers/jsonataWrapper';
+
+import { useMicrofrontendContext } from 'shared/contexts/MicrofrontendContext';
+
+export function EventList({
```

Comment:
Can't we just use our built-in `EventsList`?

Reply:
Sure we can, I did that.